src/03-setup/macos.md

```diff
@@ -5,13 +5,63 @@ All the tools can be install using [Homebrew]:
 [Homebrew]: http://brew.sh/
 
 ``` console
-$ brew cask install gcc-arm-embedded
-
 $ brew install minicom openocd
 ```
 
-If the `brew cask` command doesn't work (`Error: Unknown command: cask`), then run `brew tap
-Caskroom/tap` first and try again.
+The `gcc-arm-embedded` are no longer available through [Homebrew]. The
```

Comment:
ARM have an https://github.com/ARMmbed/homebrew-formulae which you can `brew tap ArmMbed/homebrew-formulae && brew install arm-none-eabi-gcc` that I tend to use for embedded dev and may be easier than this?